packages/react/src/components/Storage/FileUploader/FileUploader.tsx

```diff
@@ -379,6 +379,7 @@ export function FileUploader({
         isLoading={isLoading}
         isSuccessful={isSuccessful}
         hasMaxFilesError={hasMaxFilesError}
+        maxFiles={maxFiles}
```

Comment:
`maxFiles` seems too vague. How about `maxFileCount`?

Reply:
renaming `maxFiles` is a breaking change now I believe.

Reply:
This component is in dev preview, so breaking changes are allowed

Reply:
Could we separate the renaming of `maxFiles` to `maxFilesCount` into a different PR? The renaming of a prop will affect several files, and it's a bit out of scope for this PR. I'll add a ticket to do it though before GA.